src/components/Climate/index.jsx

```diff
@@ -0,0 +1,27 @@
+import React, {useState} from "react";
+import "./style.css";
+
+import temp from "./images/temp.svg";
+//proč nefunguje zobrazení obrázku, když je tato adresa napsaná přímo v kódu v html?
```

Comment:
protože v tom css se to bere vzhledem k složce /src, kdežto zde se relativní cesta bere vzhledem k tomuhle souboru. tj by tu mělo být:
```
import temp from "../../../assets/images/temp.svg";
```

Reply:
jo a nebo takhle správně protože máš podsložku img a obrázkem temp.svg tady. Ale v css se to bere vzhledem ke složce /src,